cdap-app-templates/cdap-etl/cdap-data-pipeline2_2.11/pom.xml

```diff
@@ -108,11 +108,17 @@
     <dependency>
       <groupId>io.cdap.common</groupId>
       <artifactId>common-http</artifactId>
+      <version>${cdap.common.version}</version>
```

Comment:
You should not need it. It's set with dependencty management in cdap pom.xml

Reply:
Fixed.